### DISCUSSION TITLE
How "safe" is the capacitor webview on a build release of an app (android)?

### DISCUSSION THREAD

**Discussion by distante** [ionic-team/capacitor#6013] [Q&A] [answered]
I am curious to know if someone would be able to open a chrome debugging session of the web view in a capacitor app, that was installed from the store, that is, a build release.

Would this be theoretically possibly?

**Comment by jcesarmobile** [comment, marked as answer]
It’s possible to do it if you set `WebView.setWebContentsDebuggingEnabled(true);`, but for release builds Capacitor sets it to false by default. So shouldn’t be possible unless you change the value to true on purpose.

**Reply by distante** [reply]
This is really good to know, thanks!
